src/unix/fs.c

```diff
@@ -826,7 +826,8 @@ static int uv__fs_fstat(int fd, uv_stat_t *buf) {
 
 
 typedef ssize_t (*uv__fs_buf_iter_processor)(uv_fs_t* req);
-static ssize_t uv__fs_buf_iter(uv_fs_t* req, uv__fs_buf_iter_processor process) {
+static ssize_t uv__fs_buf_iter(uv_fs_t* req, uv__fs_buf_iter_processor process,
+                               int retry_on_eintr) {
```

Comment:
It's true that it's currently always 1, but it seemed more future-proof to have the condition on 862 be exactly the same as the condition on 923. Could easily take it out if you feel it muddies the waters.